test/common/assertSnapshot.js

```diff
@@ -17,7 +17,11 @@ function replaceStackTrace(str, replacement = '$1*$7$8\n') {
 }
 
 function replaceInternalStackTrace(str) {
-  return str.replaceAll(/(\W+).*node:internal.*/g, '$1*');
+  // Replace non-internal frame `at TracingChannel.traceSync (node:diagnostics_channel:328:14)`
+  // as well as `at node:internal/main/run_main_module:33:47` with `*`.
+  str = str.replaceAll(/(\W+).*node:.*/g, '$1*');
+  str = str.replaceAll(/(\W+).*\(node:.*/g, '$1*');
```

Comment:
Is the only difference between these two patterns the `(` character? If so, shouldn't the first regex handle it? If not, couldn't the two regexes be combined?

Reply:
Thanks, merged them.